components/slot-fill/test/slot.js

```diff
@@ -16,6 +16,15 @@ import Provider from '../provider';
  */
 import { Component } from '@wordpress/element';
 
+// TODO: Avoid mock when Enzyme supports React 16.3.0+ Fragment. May require
```

Comment:
We probably should use `react-test-renderer` instead.

Reply:
And here I thought we stopped using Enzyme altogether 😬